README.md

```diff
@@ -191,6 +191,45 @@ https://github.com/viaduct-ai/kustomize-sops/issues
 
 For information, read the [kustomize generator options documentation](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/generatorOptions.md).
 
+### Generate secret directly from encrypted files
```

Comment:
Can you move this into it's own section above of the `Generator Options` section? "Generator options" is specifically referring to the kustomize generator options, not different methods of generating secrets from encrypted files.

```markdown
## Generate Secrets from Encrypted Files
```

Reply:
I have moved it to it's own section. Didn't realise I had put it in the generator options sections